mapping.md

```diff
@@ -68,21 +68,15 @@ Maps to nix.Source with `type = neo.recordingchannelgroup`.
     | RecordingChannelGroup.name(string)        | Source.name(string)                   |
     | RecordingChannelGroup.description(string) | Source.definition(string)             |
     | RecordingChannelGroup.file_origin(string) | Source.metadata(**Section**) [[1]](#notes) |
-    | RecordingChannelGroup.coordinates(Quantity 2D) | Source.metadata(**Section**) [[1]](#notes) |
 
+  - Objects
+    - For each channel in `RecordingChannelGroup`, determined by the `channel_indexes` list, a `neo.Unit` is created with `type = neo.recordingchannel`.
```

Comment:
This is a mistake. Instead of a `neo.Unit`, what is created is a `nix.Source`. Each Source represents a single Recording Channel in the group.